doc/gov/chapters/chapter09.md

```diff
@@ -81,3 +81,321 @@ Exceptions will be clearly recorded in Reference Architectures' Appendices which
+
+Prior to the beginning of field trials, the CNTT community will define an operational plan, secure resource, and provide all designated contact information required to support the trials participants.  As the results of the trails may produce data and information that could be considered sensitive by participants, CNTT will establish standard data set requirements and secure collection methods to ensure participant privacy protection.
+
+<a name="9.5.1.4"></a>
+#### 9.5.1.4 Expectations of Trials Participants
+
+Trials participants will be expected to commit to establishing an RA-1 compliant NFVi through means that are most appropriate for the participant. 
+
+First, the participant will secure appropriate environment space from pre-existing space, newly built space or securing LaaS.  The environment can exist in private or community hardware infrastructure.
+
+Second, the participant will build/setup/configure the environment space via their preferred method. This can include the use of a cookbook, automated install, and/or build from RA-1/RI-1 requirements and specifications. CNTT RI-1 Chapter 3 (https://github.com/cntt-n/CNTT/blob/master/doc/ref_impl/cntt-ri/chapters/chapter03.md ) documentation provides the matching RI-1 requirements for the build.
+
+<a name="9.5.1.5"></a>
+#### 9.5.1.5 Expectation 2: Execute the RC-1 Test suite
+
+CNTT will provide the participants with the community RC-1 Test suite, found at this Link: https://github.com/cntt-n/CNTT/blob/master/doc/ref_impl/cntt-ri/chapters/chapter04.md. The participants will execute test cases per instructions and record the quantitative results.
+
+Test case suite should be executed successfully at least 3 times. Three times represents the recommended number of test suite runs to eliminate false positives in results.  A triage process will be used to determine and manage root cause analysis of failures. If the failures are determined to be issues with the participant’s RI, CNTT will convey the issues to the RI and make available SMEs to assist the participant in resolving the issues. When failures are deemed to be caused by an issue or gap in the RA/RI/RC, the community will work to determine the resolution, and modify the RA/RI/RC accordingly.
+
+Once the test case suite execution is successful for 3 consecutive iterations, the participant will provide the data of all iterations (both successful and unsuccessful) to CNTT based on participant privacy expectations (See Expectation #4)
+
+<a name="9.5.1.6"></a>
+#### 9.5.1.6 Expectation #3: The Qualitative Survey
+
+At the conclusion of Test Case iterations, the participant will be asked to complete a qualitative survey of their experience. This survey will be used to measure the feasibility, utility, and effectiveness of the RI-1 specifications, installation/configuration methods and RC-1 Test case efficacy. The survey will use an Agile User Story format. The Table below provides an example of the survey questions:
```

Comment:
The feedbacks should go through the underlying projects implementing Cookbook or test suites. Nothing new by CNTT.